.chloggen/dinesh.gurumurthy_datadog-config-pkg.yaml

```
@@ -0,0 +1,29 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
```

```suggestion
component: pkg/datadog
```

AFAIS there is no user-facing change to datadog exporter or connector